ci-operator/step-registry/ipi/conf/ibmcloud/provisionednetwork/ipi-conf-ibmcloud-provisionednetwork-commands.sh

```diff
@@ -6,9 +6,20 @@ set -o pipefail
 
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
+vpcSubnetsFile="${SHARED_DIR}/customer_vpc_subnets.yaml"
 
-cat "${SHARED_DIR}/customer_vpc_subnets.yaml"
+cat "${vpcSubnetsFile}"
+region="${LEASED_RESOURCE}"
+declare -a zones=("${region}-1" "${region}-2" "${region}-3")
```

Comment:
Not sure if it's always such a zone list `"${region}-1" "${region}-2" "${region}-3"` for IBM Cloud regions, but I think it would be better if we can query the available zone name list in real time.